### DISCUSSION TITLE
vuex store states

### DISCUSSION THREAD

**Discussion by miguelapol** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#528] [Q&A]
la verdad no tengo idea sobre el uso de los estados, alguien que tenga mas informacion

**Comment by richirrim** [comment]
Google broh, ahí está todo y otra te recomiendo la plataforma de Andrés Núñez, el imparte cursos de Vue y ahí podrás encontrar más info. 

La plataforma es, escuelavue.es.

**Comment by BattiestWinter** [comment]
A mi este video me ayudó mucho:

https://www.youtube.com/watch?v=zJkPhjjOZ0A

Y para almacenar los estados éste te ahorra un quebradero de cabeza jaja:

https://www.youtube.com/watch?v=sgaKwGvcyIQ
